packages/cli/src/imperative/pluginLifeCycle.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import {
+    AbstractPluginLifeCycle,
+    CredentialManagerOverride,
+} from "@zowe/imperative";
+import { Constants } from "./Constants";
+
+class PluginLifeCycle extends AbstractPluginLifeCycle {
+    public postInstall(): void | Promise<void> {
```

Comment:
The abstract class defines a return type of `void | Promise<void>`, but here we know we aren't returning a promise, so we could probably be more specific about the type and just use `void`.